docs/about.md

```diff
@@ -6,6 +6,16 @@ description: Positions of Steven "Destiny" Bonnell
 slug: /
 ---
 
+## Who I am
+
+I (Steven Kenneth "Destiny" Bonnell II) am most well known for my political content online, but I started my career as a [Starcraft 2](https://en.wikipedia.org/wiki/StarCraft_II:_Wings_of_Liberty) livestreamer. Today I livestream most aspects of my life; including gaming, online debates, and canvassing for political change. Throughout my life I have always tried to ensure what I believe is consistent and to test my ideas I would argue with others. I am sincere about what I say and do so if anyone has a disagreement with me they can always join [my discord's](https://discordapp.com/invite/destiny) voice chat to have a talk while I am streaming or can email me (contact@destiny.gg).
+
+## Political advocacy
+
+Starting in [November of 2020](https://youtu.be/93leuGiUSpo), I attempted to spin up an organization to mobilize youth into direct, local political action. Our first efforts were in Georgia, where we [knocked on over 20,000](https://www.wrbl.com/news/georgia-news/math-mvmt-hosts-canvassing-event-ahead-of-senate-run-offs/) doors in support of Jon Ossoff and Reverend Warnock. For my next effort, I wanted to focus on my hometown where my 9 year old son lives to see if we could mobilize enough volunteers to get a progressive candidate elected to office. Unfortunately because of how people and news organisations [mischaracterized my actions](https://www.reddit.com/r/Destiny/comments/lx0dnw/incredibly_disappointed_at_the_lack_of/), I was forced to stop campaigning for my mayor of choice.
```

Comment:
```suggestion
Starting in [November of 2020](https://youtu.be/93leuGiUSpo), I attempted to spin up an organization to mobilize youth into direct, local political action. Our first efforts were in Georgia, where we [knocked on over 20,000](https://www.wrbl.com/news/georgia-news/math-mvmt-hosts-canvassing-event-ahead-of-senate-run-offs/) doors in support of Jon Ossoff and Reverend Warnock. For my next effort, I wanted to focus on my hometown where my 9 year old son lives to see if we could mobilize enough volunteers to get a progressive candidate elected to office. Unfortunately because of how people and news organizations [mischaracterized my actions](https://www.reddit.com/r/Destiny/comments/lx0dnw/incredibly_disappointed_at_the_lack_of/), I was forced to stop campaigning for my mayor of choice.
```